src/ast/astExtractor/expression-extractor.ts

```diff
@@ -312,6 +387,32 @@ export class ExpressionExtractor extends BaseJavaCstVisitorWithDefaults {
     return literal;
   }
 
+  booleanLiteral(ctx: BooleanLiteralCtx) {
+    return {
+      kind: "Literal",
+      literalType: {
+        kind: "BooleanLiteral",
+        value: ctx.False ? "false" : ("true" as "true" | "false"),
```

Comment:
```suggestion
        value: ctx.False ? "false" : "true",
```

Reply:
Does this not work as well?

Reply:
In the case of your suggestion, the typing of `value` does not work as your suggestion will result in `value: string` while the above is `value: "true" | "false"`. However, I wonder why the assertion only applies to the alternative expression hmm...